web/src/reducers/budget.js

```diff
@@ -328,6 +337,14 @@ const buildBudget = bigState => {
               Non-personnel:     $8:  10% of state share
       */
 
+      // Record these costs for each FFY of the activity. These are used in
+      // the cost allocation display, so the user can see at an activity
+      // level their total costs and cost distributions.
+      activityCostByFFY[year].total = totalCost;
+      activityCostByFFY[year].federal = costShares.fedShare;
+      activityCostByFFY[year].state = costShares.stateShare;
+      activityCostByFFY[year].medicaidShare = totalMedicaidShare;
```

Comment:
We were already calculating all of this, just not stashing it off in the budget object.